### PR TITLE
/play will always stop current file playing. 

### PR DESCRIPTION
**Pull request by Ola-Palm**
The current file play via /play is always stopped before new file is played. 
Empty filename will stop current file and then return.

### CODE REVIEW

File: modules/debug_cmd/debug_cmd.c [hunk 1]
```diff
@@ -84,19 +84,28 @@ static int cmd_ua_debug(struct re_printf *pf, void *unused)
...
-		return err;
+	int err = 0;
 
-	err = play_file(NULL, baresip_player(), filename, 0);
-	if (err) {
-		warning("debug_cmd: play_file(%s) failed (%m)\n",
-			filename, err);
-		return err;
+	/* Stop the current tone, if any */
+	g_play = mem_deref(g_play);
```
Comment (Encamy) [inline modules/debug_cmd/debug_cmd.c]: You are derefing a variable that was not allocated. Is this is a correct behaviour?

File: modules/debug_cmd/debug_cmd.c [hunk 2]
```diff
@@ -84,19 +84,28 @@ static int cmd_ua_debug(struct re_printf *pf, void *unused)
...
+
+	if (str_isset(filename))
```
Comment (Encamy) [inline modules/debug_cmd/debug_cmd.c]: If `filename` will not be set, this block would not change value of `err`. So you are going to return 0. But this means that the function which purpose was to play the file did not perform its action. What about returning some error code?
(It is just an idea)

**Comment by alfredh** [comment]
many thanks @Ola-Palm 

I will merge this to master after the Baresip version 0.5.12 is released.

**Comment by alfredh** [comment]
thanks @Ola-Palm !

I decided to merge it into the 0.6.0 release

**Pull request merged**